### PR TITLE
Missing readme contributions #147

### PR DESCRIPTION
**Pull request by nrowley**
added missing readme contributions as stated #147

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,255 +39,272 @@ For instance:
...
+- [Point is inside triangle check](https://github.com/williamfiset/algorithms/tree/master/src/main/java/com/williamfiset/algorithms/geometry/PointInsideTriangle.java) **- O(1)**
+- [Point rotation about point](https://github.com/williamfiset/algorithms/tree/master/src/main/java/com/williamfiset/algorithms/geometry/PointRotation.java) **- O(1)**
+- [Triangle area algorithms](https://github.com/williamfiset/algorithms/tree/master/src/main/java/com/williamfiset/algorithms/geometry/TriangleArea.java) **- O(1)**
+- [[UNTESTED] Circle-circle intersection area](https://github.com/williamfiset/algorithms/tree/master/src/main/java/com/williamfiset/algorithms/geometry/CircleCircleIntersectionArea.java) **- O(1)**
+- [[UNTESTED] Circular segment area](https://github.com/williamfiset/algorithms/tree/master/src/main/java/com/williamfiset/algorithms/geometry/CircularSegmentArea.java) **- O(1)**
+- [Minimum cost polyhgon triangulation](https://github.com/williamfiset/algorithms/tree/master/src/main/java/com/williamfiset/algorithms/geometry/MinimumCostConvexPolygonTriangulation.java) **- O(n^3)**
```
Comment (williamfiset) [inline README.md:175]: Polygon instead of Polhygon :)

**Comment by williamfiset** [comment]
Thank you!

File: README.md (hunk 1 above)
Comment (nrowley) [reply README.md:175]: Hi i will change this asap

**Comment by williamfiset** [comment]
Please fix merge conflict

**Comment by nrowley** [comment]
will do

**Pull request closed without merging**